File: API/Models/ProximaSessaoModel.cs
namespace API_Visitatus.Models
{
    public class ProximaSessaoModel
    {
        public long SesCodi { get; set; }
        public long SesNume { get; set; }
        public DateTime SesDtHr { get; set; }
        public bool SesLibe { get; set; }
        public bool SesStat { get; set; }
        public string? SesNome { get; set; }
        public int TotalPresenca { get; set; }
    }
}

File: Sistema/src/app/models/ProximaSessao.Model.ts
export class ProximaSessaoModel {
  SesCodi: number;
  SesNume: number;
  SesDtHr: Date;
  SesLibe: boolean;
  SesStat: boolean;
  SesNome: string;
  TotalPresenca: number;

  constructor() {
    this.SesCodi = 0;
    this.SesNume = 0;
    this.SesDtHr = new Date();
    this.SesLibe = false;
    this.SesStat = false;
    this.SesNome = '';
    this.TotalPresenca = 0;
  }
}
